nilearn/plotting/html_surface.py

```diff
@@ -433,9 +435,10 @@ def view_img_on_surf(
 
 @fill_doc
 def view_surf(
-    surf_mesh,
+    surf_mesh=None,
```

Comment:
Warning: this is technically a change in the API.

Reply:
Yes, but I don't think it is harmful. Maybe mention it explicitly in the changelog.